config/vector.yaml

```diff
@@ -131,6 +147,7 @@ sinks:
     type: loki
     inputs:
       - parsed_hs_server_logs
+      - local_file_logs 
```

Comment:
this won't be needed since the file logs are already present in `parsed_hs_server_logs`